test/quantization/test_quantize_script.py

```diff
@@ -2569,3 +2570,9 @@ def forward(self, x):
         FileCheck().check_count("quantized::linear_dynamic(", 2, exactly=True) \
                    .check_not("aten::_choose_qparams_per_tensor") \
                    .run(m1.graph)
+
+class TestQATScript(QuantizationTestCase):
+
+    def fake_quant_scriptable(self):
```

Comment:
I think this test case should be in https://github.com/pytorch/pytorch/blob/master/test/quantization/test_workflow_module.py instead

Reply:
you can create a similar test like https://github.com/pytorch/pytorch/blob/master/test/quantization/test_workflow_module.py#L273